@AH-64D Apache Longbow/Addons/fza_ah64_systems/functions/core/fn_coreVariables.sqf

```diff
@@ -0,0 +1,68 @@
+/* ----------------------------------------------------------------------------
+Function: fza_systems_fnc_variables
+
+Description:
+    ...
+
+Parameters:
+	_heli      - The helicopter to get information from [Unit].
+
+Returns:
+    ...
+
+Examples:
+    ...
+
+Author:
+	BradMick
+---------------------------------------------------------------------------- */
+params ["_heli"];
+#include "\fza_ah64_systems\headers\systems.hpp"
+
+//Switch states
+_heli setVariable ["fza_systems_battSwitchOn",      false];
+
+//Electrical System
+//--Battery
+_heli setVariable ["fza_systems_battPower_pct",     1.0];
+private _battTime = SYS_BATT_TIMER * 60;
+_heli setVariable ["fza_systems_battTimer",         _battTime];
+//--Buses
+_heli setVariable ["fza_systems_battBusOn",         false];
+_heli setVariable ["fza_systems_acBusOn",           false];
+_heli setVariable ["fza_systems_dcBusOn",           false];
```

Comment:
These not having a ,true means that they will not be synchronised across multiplayer. Are you sure that none of these need to be synchronised?

If any do, then having them be set in a loop will become untenable - that will cause too much of a network issue

Reply:
Oversight, fixed, they should definitely be synchronized.